content/intro-curriculum/A_Introduction.Rmd

```diff
@@ -3,14 +3,15 @@ title: "A. Introduction to R"
 author: "Jeffrey W. Hollister & Luke Winslow"
 date: "2016-07-10"
 slug: "Introduction"
+image: "img/main/intro-icons-300px/r-logo.png"
 output: USGSmarkdowntemplates::hugo
 ---
 
 ```{r setup, include=FALSE, warning=FALSE, message=FALSE}
 library(knitr)
 
 knit_hooks$set(plot=function(x, options) {
-  sprintf("<img src='../%s%s-%d.%s'/ alt='/%s'/>", 
+  sprintf("<img src='../%s%s-%d.%s'/ title='/%s'/>", 
```

Comment:
@ldecicco-USGS this did not add the title as hover text. Is that what we expected?